vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageTextItem.kt

```diff
@@ -92,27 +91,22 @@ abstract class MessageTextItem : AbsMessageItem<MessageTextItem.Holder>() {
                 it.bind(holder.messageView)
             }
         }
-        val textFuture = if (bindingOptions?.canUseTextFuture.orFalse()) {
```

Comment:
I am wondering why we coputed that before the next line... Maybe there were a good reason, I do not remember